tests/sentry/replays/scripts/test_delete_replays.py

```diff
@@ -0,0 +1,264 @@
+from __future__ import annotations
+
+import datetime
+from io import BytesIO
+from uuid import uuid4
+from zlib import compress
+
+from sentry.models.file import File
+from sentry.replays.models import ReplayRecordingSegment
+from sentry.replays.scripts.delete_replays import delete_replays
+from sentry.replays.testutils import (
+    mock_replay,
+    mock_rrweb_div_helloworld,
+    mock_segment_console,
+    mock_segment_fullsnapshot,
+    mock_segment_init,
+    mock_segment_nagivation,
+)
+from sentry.testutils.cases import ReplaysSnubaTestCase
+from sentry.utils.json import dumps_htmlsafe
+
+
+class TestDeleteReplays(ReplaysSnubaTestCase):
```

Comment:
this is for a separate PR, but I know from the [threadpool PR](https://github.com/getsentry/sentry/pull/66881) we'll eventually need `TransactionTestCase` here